src/main/java/util/ApplicationUtils.java

```diff
@@ -0,0 +1,22 @@
+package util;
+
+import org.commcare.api.persistence.SqlSandboxUtils;
+
+/**
+ * Utility methods for dealing with Applications
+ */
+public class ApplicationUtils {
+
+    public static Boolean deleteApplicationDbs(String appId) {
+        String dbPath = "dbs/" + appId;
```

Comment:
yeah might be worth it to offline. my gut is to namespace the app installs as well too